megamek/src/megamek/server/Server.java

```diff
@@ -11335,18 +11340,21 @@ public void deployTeleMissile(Entity ae, WeaponType wtype, AmmoType atype, int w
                     vPhaseReport.add(r);
                     te.heatFromExternal += 2 * missiles;
                     Report.addNewline(vPhaseReport);
-                } else if (te instanceof Tank) {
-                    boolean targetIsSupportVee = (te instanceof SupportTank)
-                                                 || (te instanceof LargeSupportTank)
-                                                 || (te instanceof SupportVTOL);
+                } else if (te instanceof GunEmplacement){
+                    int direction = Compute.targetSideTable(ae, te, called);
+                    while (missiles-- > 0) {
+                        HitData hit = te.rollHitLocation(ToHitData.HIT_NORMAL,
+                                                         direction);
+                        vPhaseReport.addAll(damageEntity(te, hit, 2));
+                    }
+                } else if ((te instanceof Tank) || te.isSupportVehicle()) {
```

Comment:
Currently only ground support vehicles return true to `isSupportVehicle()` but as part of my support vehicle work I'm bringing fixed wing support vehicles (and later airships and satellites) in line with their support vehicle nature. At that point fixed wing and others need to match here instead of their `ConvFighter` parent class.